edx_exams/apps/api/v1/views.py

```diff
@@ -388,13 +389,62 @@ def get(self, request, exam_id):
         return response
 
 
+class LatestExamAttemptView(ExamsAPIView):
+    """
+    Endpoint for the fetching a user's latest exam attempt.
+    /exams/attempt/latest
+
+    Supports:
+        HTTP GET: Get the data for a user's latest exam attempt.
+
+    HTTP GET
+    Fetches a user's latest exam attempt.
+
+    **GET data Parameters**
+        'user': The data of the user whose latest attempt we want to fetch.
+
+    **Returns**
+    {
+        'id': int (primary key),
+        'created': datetime,
+        'modified': datetime,
+        'user': User object,
+        'start_time': datetime,
+        'end_time': datetime,
+        'status': string,
+        'exam': Exam object,
+        'allowed_time_limit_mins': int,
+        'attempt_number': int,
+    }
+    """
+
```

Comment:
This class should also have `authentication_classes = (JwtAuthentication,)` added